STDERR: Unknown experiment: native-assets

```sh
cruvie@192 flutter_app % flutter run -d macos --enable-flutter-gpu
Launching lib/main.dart on macOS in debug mode...
Unhandled exception:
Exception: Failed to run flutter_scene_importer:import command in build hook (exit code 254):
STDERR: Unknown experiment: native-assets

STDOUT: 
#0      buildModels (package:flutter_scene_importer/build_hooks.dart:46:7)
#1      main.<anonymous closure> (file:///Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/hook/build.dart:6:5)
#2      build (package:native_assets_cli/src/api/build.dart:107:16)
#3      main (file:///Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/hook/build.dart:5:3)
#4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:312:33)
#5      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:193:12)

  Building assets for package:example_app failed.
  build.dart returned with exit code: 255.
  To reproduce run:
  /Users/cruvie/Documents/flutter/bin/cache/dart-sdk/bin/dart --packages=/Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/.dart_tool/package_config.json /Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/.dart_tool/hooks_runner/example_app/7ad245dadc068daf8ba9d7bb51a8ef8e/hook.dill --config=/Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/.dart_tool/hooks_runner/example_app/7ad245dadc068daf8ba9d7bb51a8ef8e/input.json
  stderr:
  Unhandled exception:
Exception: Failed to run flutter_scene_importer:import command in build hook (exit code 254):
STDERR: Unknown experiment: native-assets

STDOUT: 
#0      buildModels (package:flutter_scene_importer/build_hooks.dart:46:7)
#1      main.<anonymous closure> (file:///Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/hook/build.dart:6:5)
#2      build (package:native_assets_cli/src/api/build.dart:107:16)
#3      main (file:///Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app/hook/build.dart:5:3)
#4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:312:33)
#5      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:193:12)

  stdout:


Target dart_build failed: Error: Building native assets failed. See the logs for more details.
Failed to package /Users/cruvie/Documents/open-source/flutter_scene/examples/flutter_app.
Command PhaseScriptExecution failed with a nonzero exit code
** BUILD FAILED **

Building macOS application...                                           
Error: Build process failed
cruvie@192 flutter_app % 

```
```sh
cruvie@192 flutter_app % flutter config --list
All Settings:
  enable-web: (Not set)
  enable-linux-desktop: (Not set)
  enable-macos-desktop: (Not set)
  enable-windows-desktop: (Not set)
  enable-android: (Not set)
  enable-ios: (Not set)
  enable-fuchsia: (Not set)
  enable-custom-devices: (Not set)
  cli-animations: (Not set)
  enable-native-assets: true
  enable-dart-data-assets: (Not set)
  enable-swift-package-manager: (Not set)
  omit-legacy-version-file: (Not set)
  enable-windowing: (Not set)
  enable-lldb-debugging: (Not set)

cruvie@192 flutter_app % 
```
```sh
cruvie@192 flutter_app % flutter doctor -v
[✓] Flutter (Channel main, 3.37.0-1.0.pre-101, on macOS 15.6.1 24G90 darwin-arm64, locale zh-Hans-CN) [68.6s]
    • Flutter version 3.37.0-1.0.pre-101 on channel main at /Users/cruvie/Documents/flutter
    • Upstream repository git@github.com:flutter/flutter.git
    • Framework revision adca933962 (18 hours ago), 2025-09-12 19:19:41 -0300
    • Engine revision 3310be0332
    • Dart version 3.10.0 (build 3.10.0-195.0.dev)
    • DevTools version 2.50.0
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn
    • Feature flags: enable-web, enable-linux-desktop, enable-macos-desktop, enable-windows-desktop, enable-android, enable-ios, cli-animations, enable-native-assets, omit-legacy-version-file, enable-lldb-debugging


```

--- a/examples/flutter_app/hook/build.dart
+++ b/examples/flutter_app/hook/build.dart
@@ -1,4 +1,4 @@
-import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:hooks/hooks.dart';
 import 'package:flutter_scene_importer/build_hooks.dart';
 
 void main(List<String> args) {

--- a/examples/flutter_app/pubspec.yaml
+++ b/examples/flutter_app/pubspec.yaml
@@ -14,7 +14,7 @@ dependencies:
     sdk: flutter
   flutter_scene: ^0.9.2-0
   flutter_scene_importer: ^0.9.0-0
-  native_assets_cli: '>=0.13.0 <0.14.0'
+  hooks: ^1.0.0
   vector_math: ^2.1.4
 
 dev_dependencies:

--- a/packages/flutter_scene/hook/build.dart
+++ b/packages/flutter_scene/hook/build.dart
@@ -1,5 +1,5 @@
 import 'package:flutter_scene_importer/offline_import.dart';
-import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:hooks/hooks.dart';
 
 import 'package:flutter_gpu_shaders/build.dart';
 

--- a/packages/flutter_scene/pubspec.yaml
+++ b/packages/flutter_scene/pubspec.yaml
@@ -22,9 +22,7 @@ dependencies:
     sdk: flutter
   flutter_gpu:
     sdk: flutter
-  # TODO: switch to a published version (^0.4.0) once flutter_gpu_shaders 0.4.0 ships.
-  flutter_gpu_shaders:
-    path: ../../../flutter_gpu_shaders
+  flutter_gpu_shaders: ^0.4.0
   flutter_scene_importer: ^0.9.0-0
   hooks: ^1.0.0
   vector_math: ^2.1.4

--- a/packages/flutter_scene/pubspec.yaml
+++ b/packages/flutter_scene/pubspec.yaml
@@ -22,9 +22,11 @@ dependencies:
     sdk: flutter
   flutter_gpu:
     sdk: flutter
-  flutter_gpu_shaders: ^0.3.0
+  # TODO: switch to a published version (^0.4.0) once flutter_gpu_shaders 0.4.0 ships.
+  flutter_gpu_shaders:
+    path: ../../../flutter_gpu_shaders
   flutter_scene_importer: ^0.9.0-0
-  native_assets_cli: '>=0.13.0 <0.14.0'
+  hooks: ^1.0.0
   vector_math: ^2.1.4
 
 dev_dependencies:

--- a/packages/flutter_scene_importer/hook/build.dart
+++ b/packages/flutter_scene_importer/hook/build.dart
@@ -1,7 +1,7 @@
 import 'dart:io';
 
+import 'package:hooks/hooks.dart';
 import 'package:logging/logging.dart';
-import 'package:native_assets_cli/native_assets_cli.dart';
 
 void main(List<String> args) async {
   await build(args, (config, output) async {

--- a/packages/flutter_scene_importer/lib/build_hooks.dart
+++ b/packages/flutter_scene_importer/lib/build_hooks.dart
@@ -1,6 +1,6 @@
 import 'dart:io';
 
-import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:hooks/hooks.dart';
 
 void buildModels({
   required BuildInput buildInput,
@@ -26,12 +26,11 @@ void buildModels({
     outputFileName =
         '${outputFileName.substring(0, outputFileName.lastIndexOf('.'))}.model';
 
-    /// dart --enable-experiment=native-assets run flutter_scene_importer:import \
-    ///      --input <input> --output <output> --working-directory <working-directory>
+    /// dart run flutter_scene_importer:import \
+    ///     --input <input> --output <output> --working-directory <working-directory>
     final importerResult = Process.runSync(
       dartExec.toFilePath(),
       [
-        '--enable-experiment=native-assets',
         'run',
         'flutter_scene_importer:import',
         '--input',

--- a/packages/flutter_scene_importer/pubspec.yaml
+++ b/packages/flutter_scene_importer/pubspec.yaml
@@ -21,8 +21,8 @@ dependencies:
     sdk: flutter
   flutter_gpu:
     sdk: flutter
+  hooks: ^1.0.0
   logging: ^1.2.0
-  native_assets_cli: '>=0.13.0 <0.14.0'
   vector_math: ^2.1.4
 
 dev_dependencies:

